packages/cli/package.json

```diff
@@ -105,7 +105,6 @@
     }
   },
   "devDependencies": {
-    "@antfu/ni": "0.21.12",
```

Comment:
ℹ️  Dropping `ni` here as we were anyways not really using its capabilities and basically require `npm` to be present.